RecoDataProducts/inc/TrkStrawHitSeed.hh

```diff
@@ -9,67 +9,142 @@
+    auto sensorTOCA() const { return _stoca; }
+    auto fitDOCA() const { return _udoca; }
+    auto fitDOCAVar() const { return _udocavar; }
+    auto fitDt() const { return _udt; }
+    auto fitTOCAVar() const { return _utocavar; }
+    auto refDOCA() const { return _rdoca; }
+    auto refDOCAVar() const { return _rdocavar; }
+    auto refDt() const { return _rdt; }
+    auto reTOCAVar() const { return _rtocavar; }
+    auto refPOCA_Upos() const { return _rupos; }
+    auto driftRadius() const { return _rdrift; }
+    auto radialErr() const { return _rerr; }
     HitT0 const&  t0() const { return _t0; }
     Float_t trkLen() const { return _trklen; }
     Float_t hitLen() const { return _hitlen; }
-    Float_t driftRadius() const { return _rdrift; }
     Float_t signalTime() const { return _stime; }
-    Float_t hitTime() const { return _htime; }
-    Float_t TOTDriftTime() const { return _dtime; }
-    Float_t wireDOCA() const { return _wdoca; }
-    Float_t radialErr() const { return _rerr; }
-    Float_t     wireDist() const { return _wdist; }
-    Float_t     wireRes() const { return _werr; }
-    Float_t     energyDep() const { return _edep; }
-    StrawEnd    driftEnd() const { return _end; }
+    Float_t wireDOCA() const { return _rdoca; }
     Int_t ambig() const { return _ambig; }
-    StrawHitFlag const& flag() const { return _flag; }
+    //
+    //  Payload
     //
     StrawHitIndex   _index;       // index to the original straw (Combo) hit, and (for MC) MCDigi
-    StrawId     _sid;   // which straw has the hit
-    HitT0     _t0;    // time origin for this hit = track t0 + particle propagation time to this straw
-    Float_t     _trklen;    // Length from the nominal track start to the POCA of this hit
-    Float_t     _hitlen;    // Length from the straw center to the POCA of this hit
-    Float_t     _rdrift;    // drift radius for this hit
-    Float_t     _dtime;   // drift time from TOT for this hit
-    Float_t     _stime;   // signal propagation time for this hit, to the nearest end
-    Float_t     _htime;   // measured time for this hit
-    Float_t     _wdoca;   // DOCA from the track to the wire, signed by the angular momentum WRT the wire
-    Float_t     _rerr;    // intrinsic radial error
-    Int_t     _ambig;   // LR ambiguity assigned to this hit
-    Float_t         _edep;        // reco energy deposition
-    Float_t         _wdist;       // wire distance
-    Float_t         _werr;        // wire distance error estimate
-    StrawEnd      _end;         // straw end used for hit time measurement
+    StrawId         _sid;   // which straw has the hit
+    StrawEnd        _end;         // straw end used for hit time measurement
     StrawHitFlag    _flag;    // flag describing the status of this hit (active, ....)
+    Int_t           _ambig;   // hit state, including LR ambiguity
+    Int_t           _algo;     // hit updater algorithm
+    Bool_t          _frozen; // hit state was frozen
```

Comment:
_frozen is not used anywhere.  It has no accessor and it's not a c'tor argument.  I think it's safe to delete it.

Reply:
_frozen is used (in TrkAna) and is set (non-trivially) on line 42.

Reply:
Thanks.  I had forgotten that these are structs with public data, so accessors are not needed.   I see that's the same answer for all of the other things that I thought were not useful.